management/schemas/campaign.yaml

```diff
@@ -58,27 +58,29 @@ schemas:
       CustomFieldsJson:
         type: string
         nullable: true
-      StartDate:
-        type: string
-        default: 1/1/2018
-        deprecated: true
-      EndDate:
-        type: string
-        nullable: true
-        deprecated: true
       IsFreqCap:
         type: boolean
         nullable: true
         readOnly: true
 
+  CampaignInput:
+    allOf:
+      - $ref: '#/schemas/CampaignBase'
+      - type: object
+        properties:
+          Flights:
+            type: array
+            items:
+              type: object
+            nullable: true
+
   Campaign:
     allOf:
       - $ref: '#/schemas/CampaignBase'
       - type: object
-        required:
-          - Id
-          - StartDate
         properties:
+          required:
```

Comment:
Is this location correct? `required` previously used to be at the top level, but now it is underneath `properties`

Reply:
Thanks for flagging this one!